### DISCUSSION TITLE
Download hyprland terminal emulator

### DISCUSSION THREAD

**Discussion by Moorocks** [hyprwm/Hyprland#6012] [Q&A]
I cannot use hyprland as my laptop config is too low, but i love the terminal emulator. 
Can anyone suggest me how to get the terminal emulator of hyprland.

**Comment by fufexan** [comment]
There's no terminal specific to Hyprland. Most people use Kitty though, if that's what you're asking.

**Reply by Moorocks** [reply]
Umm I thought the same too, but it doesn't seem like it. When i tried to install hyprland and discontinued the setup halfway stage, my terminal had a different view. Please refer to the below image for more info.
![image](https://github.com/hyprwm/Hyprland/assets/143619572/daff2caa-95da-43b9-afdf-9ac8ef6c4797)

I do not think this is kitty terminal, must be some other customized terminal setup.
